piptools/scripts/compile.py

```diff
@@ -61,6 +67,12 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Clear any caches upfront, rebuild from scratch",
 )
+@click.option(
+    "-e",
```

Comment:
```suggestion
    "-E",
```

I'would like to suggest `-E` instead of `-e`, which clashes with `pip install -e`.

Reply:
Good catch. Let's just drop the short key and leave only `--extras`

Reply:
Fixed :+1: